examples/pip_parse_vendored/BUILD.bazel

```diff
@@ -16,7 +16,7 @@ genrule(
     cmd = " | ".join([
```

Comment:
So I hate to ask this, but can we do this with starlark and not bash?  A lot of windows users don't have bash installed.

Reply:
No, please see #979 for reasons why.

Reply:
I'd like to leave this out of scope of this PR.